cli/tests/unit/net_test.ts

```diff
@@ -985,6 +985,41 @@ Deno.test(
   },
 );
 
+Deno.test(
+  { permissions: { read: true, run: true, net: true } },
+  async function netUdpListenUnrefAndRef() {
+    const p = execCode2(`
```

Comment:
Please test the Node.js API instead of `Deno.listenDatagram`. It can be written in `cli/tests/unit_node/dgram_test.ts`